exp-models/addon/models/account.js

```diff
@@ -23,5 +22,10 @@ export default DS.Model.extend(JamModel, {
         };
 
         return profiles.filter(getProfile)[0];
+    },
+    pastSessionsFor(experiment) {
+        return this.get('store').queryRecord(experiment.get('sessionCollectionId'), {
+            q: `profileId:${this.get('id')}*`
```

Comment:
For Lookit, you rarely want to query by different profiles, but you often want to group by different profiles. But for any given route, I think you will almost always want all the past sessions for the whole account.

One example is the past studies page, where you're listing the previous sessions and the feedback. You'd have every previous session listed regardless of which child took the session. For the longitudinal studies, you'd want to be able to calculate which child is in which state, but you'd need to do that for every child at the same time.

That being said, I may be missing a detail of implementation, so feel free to ignore.